sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -700,13 +696,10 @@ public long count(CaseCriteria caseCriteria) {
 				caseRoot.get(Case.CASE_IDENTIFICATION_SOURCE),
 				caseRoot.get(Case.SCREENING_TYPE),
 				// responsible jurisdiction
-				joins.getResponsibleRegion().get(Region.UUID),
 				joins.getResponsibleRegion().get(Region.NAME),
-				joins.getResponsibleDistrict().get(District.UUID),
 				joins.getResponsibleDistrict().get(District.NAME),
-				joins.getResponsibleCommunity().get(Community.UUID),
-				joins.getResponsibleCommunity().get(Community.NAME)
-				);
+				joins.getResponsibleCommunity().get(Community.NAME),
+				caseService.jurisdictionSelector(cb, caseService.inJurisdictionOrOwned(cb, joins)));
```

Comment:
Should we rename the property in the Case..Dto to inJurisdictionOrOwned? If not we should at least add a comment to the getter to point to CaseService.inJurisdictionOrOwned